scripts/lcov.sh

```diff
@@ -30,9 +30,15 @@ EOF
 
 set -eu
 
-# Repository detection
-in_mbedtls_build_dir () {
-    test -d library
+# Project detection
+PROJECT_NAME_FILE='./scripts/project_name.txt'
+if read -r PROJECT_NAME < "$PROJECT_NAME_FILE"; then :; else
+    echo "$PROJECT_NAME_FILE does not exist... Exiting..." >&2
+    exit 1
+fi
+
+in_mbedtls_repo () {
```

Comment:
This is missing a change compared to #8604: shouldn't the call to `in_mbedtls_build_dir` below change to `in_mbedtls_repo`?

Reply:
Typo there, it's #8609, and you are completely right. I really thought I had changed that though which is odd but I will do now.